src/components/BooksForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const categoryList = [
+  'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi',
+];
+
+const BooksForm = () => (
```

Comment:
Improve the file structure: `BooksForm` should be placed in `src/containers` directory.

![image](https://user-images.githubusercontent.com/31897434/90568251-6ba33b80-e1a3-11ea-990d-e73db732b31b.png)
[Microverse Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/)